lib/community/widgets/post_card_metadata.dart

```diff
@@ -353,6 +353,78 @@ class UrlPostCardMetaData extends StatelessWidget {
   }
 }
 
+class CrossPostMetaData extends StatelessWidget {
+  final int unreadComments;
+  final int comments;
+  final bool hasBeenEdited;
+  final DateTime published;
+  final bool saved;
+  final int score;
+
+  const CrossPostMetaData({
+    super.key,
+    required this.unreadComments,
+    required this.comments,
+    required this.hasBeenEdited,
+    required this.published,
+    required this.saved,
+    required this.score,
+  });
+
+  final MaterialColor upVoteColor = Colors.orange;
+  final MaterialColor downVoteColor = Colors.blue;
+  final MaterialColor savedColor = Colors.purple;
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+
+    return BlocBuilder<ThunderBloc, ThunderState>(
+      builder: (context, state) {
+        return Row(
+          mainAxisSize: MainAxisSize.min,
+          children: [
+            IconText(
```

Comment:
Are we able to replace this to use `ScorePostCardMetaData` from `post_card_metadata.dart`? This would help reduce duplicate logic.

If we need to apply a dimmer look to the content, we could possibly set `hasBeenRead` to true when using `ScorePostCardMetaData`. It might not be exactly the same but could be close enough. Alternatively, we could introduce a new `opacity` parameter to `ScorePostCardMetaData` which overrides the existing opacity.

Reply:
Good idea